packages/drivers/http/README.md

```diff
@@ -0,0 +1,28 @@
+# @k-ramel/driver-http
+> HTTP driver for k-ramel
+
+ - Main repository: [k-ramel](https://github.com/alakarteio/k-ramel)
+
+This driver is added by default to k-ramel stores.
```

Comment:
I may have say it in another PR but why this driver is added by default ?

Reply:
Yeah I am still septical.

The purpose of k-ramel is to have a lightweight API that works 90% of the time.
HTTP is used 90% of the time I think: I want to add it per default so the user doesn't have to:
 1. Import it
 2. Add it to the second parameter of `createStore`

BUT BUT BUT, And this is all about JS ecosystem... By adding it by default, the bundle.js is larger, and this is where I am septical.

Reply:
I'm in the other 10% with conference-hall 😄 

But, I don't think it's necessary to include it by default, because it's really not painful to import it. Maybe, it could be imported in each example of the documentation, and it will not be missed.

Reply:
hmm it bother me.